src/models/Model.php

```diff
@@ -24,4 +28,31 @@ protected static function getMc(): Memcached {
     }
     return self::$mc;
   }
+
+  protected static function setMCRecords(string $key, mixed $records): void {
+    $mc = self::getMc();
+    $mc->set(
+      static::$MC_KEY.static::$MC_KEYS->get($key),
+      $records,
+      static::$MC_EXPIRE,
+    );
+  }
+
+  protected static function getMCRecords(string $key): mixed {
+    $mc = self::getMc();
+    $mc_result = $mc->get(static::$MC_KEY.static::$MC_KEYS->get($key));
+    /* HH_IGNORE_ERROR[4110] */
```

Comment:
We probably haven't been doing this enough, but if we're going to use an `HH_IGNORE`, let's specify why we're ignoring it in the comment, i.e. `/* HH_IGNORE_ERROR[4110]: xyz */`